README.md

```diff
@@ -178,8 +199,7 @@ We use Sendgrid to send mails, and you need a Sendgrid API to make it work. Addi
 `config.SMTP_PASSWORD` | (Falls back to config file)
 
 ### Upload
-We use AWS S3 services for storing the zip file of the generated event. These zip files are automatically deleted after 3 days from the store. Each
-uploaded file has a unique download link. `node-uuid` module is used for generating the cryptic download link of the file.
```

Comment:
Lets keep the points instead of adding a paragraph & improve the text

Reply:
I did'nt got that ! Can you explain ? @geekyd

Reply:
I meant that we should keep using the points here instead of paragraph.

Reply:
@geekyd Can you please comment out the area, I'm not getting which point are you talking about.
I'm sorry for the inconvenience 😖